src/service/action-impl.js

```diff
@@ -200,13 +211,25 @@ export class ActionService {
         this.addChangeDetails_(event);
         this.trigger(dev().assertElement(event.target), name, event);
       });
+    } else if (name == 'input-debounced') {
+      const debouncedInput = debounce(this.ampdoc.win, event => {
+        const target = dev().assertElement(event.target);
+        this.trigger(target, name, /** @type {!ActionEventDef} */ (event));
```

Comment:
similar to`change` event we want to include the `value` of the input at the time of trigger as `details` on the event object. Maybe require some refactoring of `addChangeDetails_` to share code.